app/js/main.js

```diff
@@ -140,6 +140,33 @@ function startFlowAuto() {
     });
     return;
   }
+  if (flow === 'updateSubscription') {
```

Comment:
Can the method comment above be updated wrt to current valid values?

Reply:
Ah, yes, thank you for catching that!